.env

```
@@ -3,15 +3,22 @@ NODE_ENV=development
 PORT=3000
 HOST=localhost
 
-# Placeholder values for required environment variables
-SUPABASE_URL=https://placeholder.supabase.co
-SUPABASE_KEY=placeholder_anon_key
-SUPABASE_SERVICE_KEY=placeholder_service_key
-JWT_SECRET=placeholder_jwt_secret_at_least_32_characters_long_for_development
+# Required environment variables (set via repository secrets)
+SUPABASE_URL=${SUPABASE_URL}
+SUPABASE_KEY=${SUPABASE_ANON_KEY}
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)